# coding:utf-8

# this program is used to throw exception.

ls = [1, 2, 3, 4]

print(ls[-1])

ls.clear()

# IndexError: list index out of range
print(ls[0])
